_posts/commands/2020-05-01-mom_prog_reset.md

```diff
@@ -5,4 +5,4 @@ tags:
   - progress
 ---
 
-Resets progress (clears in progress.dat).
\ No newline at end of file
+Resets progress (clears progress.dat).
```

Comment:
This is actually an internal campaign command, and probably won't exist by 2.0.0.

Reply:
Ya there are a few like this, like the one for testing out the stage completion code. I tagged these as debug for now. Should i remove those?

Reply:
Yep.